libs/ui/tooltip/helm/src/lib/hlm-tooltip.directive.ts

```diff
@@ -0,0 +1,41 @@
+import { Directive, inject } from '@angular/core';
+import { BrnTooltipDirective } from '@spartan-ng/ui-tooltip-brain';
+
+@Directive({
+	selector: '[hlmTooltip]',
+	standalone: true,
+	hostDirectives: [
+		{
+			directive: BrnTooltipDirective,
+			inputs: [
+				'brnTooltip: hlmTooltip',
+				'aria-describedby',
+				'disabled',
+				'position',
+				'positionAtOrigin',
+				'hideDelay',
+				'showDelay',
+				'exitAnimationDuration',
+				'touchGestures',
+			],
+		},
+	],
+})
+export class HlmTooltipDirective {
+	private readonly _brnTooltip: BrnTooltipDirective = inject(BrnTooltipDirective, { host: true });
+
+	constructor() {
+		if (this._brnTooltip) {
+			this._brnTooltip.exitAnimationDuration = 150;
+			this._brnTooltip.hideDelay = 300;
+			this._brnTooltip.showDelay = 150;
+			this._brnTooltip.tooltipContentClasses =
+				'overflow-hidden rounded-md border border-border bg-popover px-3 py-1.5 text-sm text-popover-foreground shadow-md fade-in-0 zoom-in-95 ' +
+				'data-[state=open]:animate-in ' +
+				'data-[state=closed]:animate-out data-[state=closed]:fade-out-0 data-[state=closed]:zoom-out-95 ' +
+				'data-[side=below]:slide-in-from-top-2 data-[side=above]:slide-in-from-bottom-2 ' +
+				'data-[side=left]:slide-in-from-right-2 data-[side=right]:slide-in-from-left-2 ' +
+				'data-[side=after]:slide-in-from-left-2 data-[side=before]:slide-in-from-right-2 ';
```

Comment:
How does this behave in a right-to-left layout? 
Is this handled by tailwind or is it sliding from the "wrong" side?

Reply:
you have a good point here too. I'll need to address this

Reply:
@elite-benni if you have time to tackle this feel free to create a PR. I am a little tied up right now and won't get to it in the next couple weeks...

Reply:
I just added an PR, just removing the todo and `data-[side=after]` classes.
First I thought I just have to add `rtl:` selector for this.
As I tried out the solution i found out, that it is actually not a problem. 
`before` and `after` are translated to `left` and `right` with taking the rtl direction into account. 
Everything was fine. ;)